### PR TITLE
fix: bookmark loops popup image and streaks cta

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- As the title says.

Screenshot 2024-04-25 at 8 33 56 AM

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### Preview domain
https://fix-bookmark-loops-image.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/modals/generic/common.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const promotion: Record<string, MarketingCta> = {
       description:
         'Your saved posts are waiting for you on daily.dev mobile. Perfect for reading anytime, anywhere.',
       image: cloudinary.promotions.bookmarkLoops,
-      ctaText: 'Install the app',
+      ctaText: 'Tell me more',
```
Comment (rebelchris) [inline packages/shared/src/components/modals/generic/common.ts:33]: This is not in figma where was this change decided?

Comment (sshanzel) [reply packages/shared/src/components/modals/generic/common.ts:33]: This is for streaks. Kindly roll up the file to see the parent object.

Comment (sshanzel) [reply packages/shared/src/components/modals/generic/common.ts:33]: Oh, wait it should be the other object.

**Comment by sshanzel** [comment]
Screenshot 2024-04-25 at 12 47 16 PM
Screenshot 2024-04-25 at 12 46 48 PM

**Review by rebelchris — APPROVED** [review]

**Pull request merged**